lib/Watcher.js

```diff
@@ -43,7 +43,7 @@ function Watcher(options) {
             ignored: normalizedIgnorePatterns,
             cwd: dir,
             persistent: (options.persistent !== false),
-            usePolling: false,
+            usePolling: (options.usePolling !== false),
```

Comment:
Shouldn't this be the following?:

```javascript
usePolling: options.usePolling
```

This would allow the default behavior to be used unless a value is provided for `options.usePolling` .